pkg/autostart/io.lima-vm.autostart.INSTANCE.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>io.lima-vm.{{ .INSTANCE }}</string>
```

Comment:
I would like to add an additional subdomain here:

```suggestion
	<string>io.lima-vm.autostart.{{ .INSTANCE }}</string>
```

Just to reserve part of the namespace if we want to create launch agents for other purposes in the future. I know we reserved instance names starting with `_`, so have some reserved space, but adding the `autostart` "subdomain" feels cleaner.